middleware/csrf.go

```diff
@@ -61,7 +61,17 @@ type (
 		// Indicates SameSite mode of the CSRF cookie.
 		// Optional. Default value SameSiteDefaultMode.
 		CookieSameSite http.SameSite `yaml:"cookie_same_site"`
+
+		// ErrorHandler defines a function which is executed for returning custom errors.
+		ErrorHandler CSRFErrorHandler
```

Comment:
let it be of type `func(err error, c echo.Context) error` and remove `ErrorHandlerWithContext` field.  Middlewares with 2 handlers are because first handler was added without context and due backwards compatibility only way to add context we need to introduce new field.

Reply:
Done